config/environments/development.rb

```diff
@@ -38,4 +38,10 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.action_mailer.default_url_options = { host: 'localhost:3000' }
+  
+  config.action_mailer.delivery_method = :smtp
```

Comment:
No need for this delivery_method config line - it's specified in `initializers/mail.rb` for the development environment.